packages/eslint-plugin/src/types/eslint.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module 'eslint/lib/rules/utils/keywords' {
```

Comment:
Could go away if we toss `eslint` dependency.

Reply:
Can you please advise whether we can avoid this file.
The fact that I've seen none if this kind in the repo makes me think that there is another way.

Reply:
Moved to `typings/eslint-keyword.d.ts`.